### PR TITLE
Fix tldrwl triggers; support separate api key

### PR DESCRIPTION
**Pull request by jrodal98**
## What

- `.lower()` all of the triggers
- Support setting an additional TLDRWL_API_KEY

## Why

- Bugfix
- The free OpenAI accounts do not play nicely with tldrwl due to aggressive rate limiting. However, one may want to take advantage of the free accounts for everything else, while they still can.

### CODE REVIEW

File: dicebot/handlers/message/tldrwl_handler.py [hunk 1]
```diff
@@ -9,17 +9,20 @@
...
-    "tldl",
-    "tldrwl",
-    "tl;dr",
-    "tl;dw",
-    "tl;dl",
-    "tl;drwl",
-    "bro, I don't have time for this shit",
-)
+TLDRWL_TRIGGERS = [
+    t.lower()
```
Comment (logan-stytch) [inline dicebot/handlers/message/tldrwl_handler.py:13]: This is so ugly lol

File: dicebot/app/bot.py [hunk 2]
```diff
@@ -2,12 +2,15 @@
...
 import asyncio
 import logging
+import os
 import sys
 
 from dicebot.app import engine
 from dicebot.core.client import Client
 from dicebot.logging.colored_log_formatter import ColoredLogFormatter
 
+from tldrwl.register import Register
```
Comment (logan-stytch) [inline dicebot/app/bot.py:12]: third party libs should go above dicebot libs, right?

**Review by logan-stytch — CHANGES_REQUESTED** [review]

File: dicebot/app/bot.py (hunk 2 above)
Comment (jrodal98) [reply dicebot/app/bot.py:12]: ah yeah, I spedrun and read it as discord and was making it alphabetical. I'll fix it

File: dicebot/handlers/message/tldrwl_handler.py (hunk 1 above)
Comment (jrodal98) [reply dicebot/handlers/message/tldrwl_handler.py:13]: 😡

**Pull request merged**